packages/scalecube-services/src/services/ServiceCall.js

```diff
@@ -45,7 +45,7 @@ export class ServiceCall {
                 thisMs: this.microservices,
                 meta: source.inst.service.meta || source.inst.service.constructor.meta || {}
             }))
-            .pipe(source$ => this.microservices.preRequest(source$))
+            .pipe(source$ => Observable.from(this.microservices.preRequest(source$)))
```

Comment:
@dmitriyslaym can you explain why do you wrap source$ with Observable.from? source$ is a stream and it's sending into preRequest hook and it's expected to receive stream from the hook.